DEVELOPMENT.md

```diff
@@ -9,6 +10,14 @@ Sheriff consists of three processes:
 
 The entry point is always the `init` function.
 
+# Development
+
+## Setup
+We are using `yarn` as our package manager. To install all dependencies, run the following command:
```

Comment:
Since `yarn` is mentioned here. Do you think it would make sense to move to pnpm?

Reply:
disclaimer: I am not too deep into the pros and cons of single package managers.

`pnpm` is probably the more modern solution which offers some benefits. However I think that these benefits would not have a significant improvement over `yarn` in our case here.

I did not research it yet but I am not sure if there's a good replacement for `yalc` or if `pnpm` could do the same out of the box.

Reply:
In our other projects we moved already to pnpm. I don't expect that pnpm can replace yalc